packages/suite/src/views/wallet/send/components/Outputs/components/Amount/index.tsx

```diff
@@ -138,7 +138,7 @@ const Amount = ({ output, outputId }: Props) => {
     }
 
     const withTokens = hasNetworkFeatures(account, 'tokens');
-    const symbolToUse = areSatsDisplayed ? 'sats' : symbol.toUpperCase();
+    const symbolToUse = areSatsDisplayed ? 'sat' : symbol.toUpperCase();
```

Comment:
This whole commit could be in separate PR e27460095ef5a95cf7a47763f8070a441d89efd6

Reply:
https://github.com/trezor/trezor-suite/pull/5923